tools.go

```diff
@@ -0,0 +1,12 @@
+// +build tools
+
+package tools
+
+import (
+	"github.com/UnnoTed/fileb0x"
+	"github.com/cespare/reflex"
+	"github.com/haya14busa/goverage"
+	"github.com/kballard/go-shellquote"
+	"github.com/ogier/pflag"
+	"github.com/restic/calens"
```

Comment:
I think these are intended to be blank imports. Also, I think this is meant for go-tools (https://github.com/golang/tools) not sure if it is intended to work with any package, as per the thread reads. Need to do more digging imho

Reply:
I also see more benefits when it comes with a tool that generates code, as different versions of the tool will yield different results

Reply:
I see your points but I find it a bit anoying that my tooling (vscode, make) is removing those dependencies from the go.mod file and I have to put them back manually.
Maybe I need to change something in my tooling or I'm the only one anoyed by that.

If it is like that I could also live with just closing this PR.

Reply:
I'm okay with it :)